pkg/controller/installation/core_controller.go

```diff
@@ -600,11 +602,32 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 
 	// Write the discovered configuration back to the API. This is essentially a poor-man's defaulting, and
 	// ensures that we don't surprise anyone by changing defaults in a future version of the operator.
+	// Note that we only write the 'base' installation back. We don't want to write the changes from 'overrides', as those should only
+	// be stored in the 'overrides' resource.
 	if err := r.client.Update(ctx, instance); err != nil {
 		r.SetDegraded("Failed to write defaults", err, reqLogger)
 		return reconcile.Result{}, err
 	}
 
+	// update Installation with 'overlay'
+	overlay := operator.Installation{}
+	if err := r.client.Get(ctx, utils.OverlayInstanceKey, &overlay); err != nil {
+		if !apierrors.IsNotFound(err) {
+			reqLogger.Error(err, "An error occurred when querying the 'overlay' Installation resource")
+			return reconcile.Result{}, err
+		}
+		reqLogger.V(5).Info("no 'overlay' installation found")
+	} else {
+		instance.Spec = overrideInstallationSpec(instance.Spec, overlay.Spec)
+		reqLogger.V(2).Info("loaded final override config", "config", instance)
```

Comment:
```suggestion
		reqLogger.V(2).Info("loaded computed config", "config", instance)
```
I don't think we refer any place else to this as being an override so we should be consistent and use overlay or computed and I think computed is probably best here.